microsoft-edge/devtools-guide-chromium/whats-new/includes/edge-whats-new-note.md

```diff
@@ -3,6 +3,10 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.prod: microsoft-edge
 ms.topic: include
-ms.date: 02/12/2021
+ms.date: 05/13/2021
```

Comment:
@zoherghadyali  - fyi in the next What's New, I'll ask you to review this wording for top of every What's New.